File: p2942.java
class Solution {
    public List<Integer> findWordsContaining(String[] words, char x) {
        ArrayList<Integer> indicesArray = new ArrayList<Integer>();

        for (int i = 0; i < words.length; i++){
            if (words[i].contains(x + "")){ // words[i].indexOf(x) != -1
                indicesArray.add(i);
            }
        }

        return indicesArray;
    }
}

File: p2942.py
class Solution(object):
    def findWordsContaining(self, words, x):
        """
        :type words: List[str]
        :type x: str
        :rtype: List[int]
        """

        indices_array = []

        for index, word in enumerate(words):
            if x in word:
                indices_array.append(index)


        return indices_array